src/router.tsx

```diff
@@ -30,8 +30,8 @@ import NotFound from './pages/NotFound.tsx'
 const router = createBrowserRouter(
   createRoutesFromElements(
     <>
-      <Route path="confirmations/:token" element={<Confirmation />} />
-      <Route path="password_resets/:token" element={<PasswordResets />} />
+      <Route path="auth/confirmations/:token" element={<Confirmation />} />
```

Comment:
As a slightly more flexible backwards compatibility patch, I think you can just redirect the new routes to the old ones. So instead of modifying the existing routes you'd add these:
```jsx
<Route path="auth/confirmations/:token" element={<Redirect to="/confirmations/:token" />} />
<Route path="auth/password_resets/:token" element={<Redirect to="/password_resets/:token" />} />
```